src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title, body, user, comment,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          {user && <UserInfo user={user} />}
+        </p>
```

Comment:
It will be better to move `posted by` inside your UserInfo. Because if we don't pass any user, in page we will see only `posted by` withoout any user name. Or you can add as default value - `posted by unknown`